scrapers/VirtualPee.yml

```diff
@@ -0,0 +1,31 @@
+name: virtualpee
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - virtualpee.com
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title:
+        selector: //h2[@class="video_title"]/text()
+        postProcess:
+          - replace:
+              - regex: "with"
+                with: ""
```

Comment:
minor nitpick the selector matches the comma as a second result (it is ignored by stash)
i would use something like this ? 
```yaml
        selector: //h2[@class="video_title"]/text()[1]
        postProcess:
          - replace:
              - regex: '\swith.*$'
                with: ""
```
to make sure that everything after " with" is removed from the title

Reply:
Sorted :-)